patches/server/0013-Paper-Plugins.patch

```diff
@@ -960,8 +960,8 @@ index 0000000000000000000000000000000000000000..ecd11d3bd6c7e80a9b33cb8c2d7ed8a5
 +    public void init(JavaPlugin plugin) {
 +        PluginMeta config = this.configuration;
 +        PluginDescriptionFile pluginDescriptionFile = new PluginDescriptionFile(
-+            config.getName(),
 +            config.getDisplayName(),
++            config.getName(),
```

Comment:
I disagree with this change. the rawName and name parameters should be set correctly. the `displayName` is not correct here. The first parameter should just be getName(), the second parameter should be getName() but replacing `_` with ` ` (which is what getDisplayName() used to be).